### PR TITLE
[DRAFT] Move to go modules, correct test issues, other fixes.

### PR DESCRIPTION
**Pull request by chriscoffee**
I'm marking this as work in progress. Hope you don't mind whilst I get the app working on 1.13 correctly.

I'll write a better description when I'm not super tired from Ghent 🇧🇪

---
This change is [Reviewable](https://reviewable.io/reviews/devopsdays/devopsdays-cli/133)

### CODE REVIEW

**Comment by mattstratton** [comment]
Thanks, @chriscoffee ! On this repo we don't mind draft PR's, etc :)

File: README.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
+ [![Go Report Card](https://goreportcard.com/badge/github.com/devopsdays/devopsdays-cli)](https://goreportcard.com/report/github.com/devopsdays/devopsdays-cli) [![GoDoc](https://godoc.org/github.com/devopsdays/devopsdays-cli?status.svg)](http://godoc.org/github.com/devopsdays/devopsdays-cli) [![GitHub release](https://img.shields.io/github/release/devopsdays/devopsdays-cli.svg)](https://github.com/devopsdays/devopsdays-cli/releases) [![MIT License](http://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)<br />
```
Comment (mattstratton) [inline README.md]: The README is dynamically generated at release, so these changes will be over-written. Changes should be made instead to `README.e.md`

Comment (chriscoffee) [reply README.md]: Oh yeah! I forgot about that 👀 I know you mentioned it when we chatted. I'll sort 👍 Need to remind myself to move the tests back to using Ponyville too!

**Comment by mattstratton** [comment]
One thing that would be nice - I noticed you are working on the dockerfile, which I think is connected to an existing issue (#109). If you add [closing keywords](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords) to your initial description of the PR, it will automatically close that issue when this PR is merged. Not sure if this PR fixes any other issues, but if so, maybe add them too?

**Comment by mattstratton** [comment]
Sorry for all the nagging bots that are critiquing your code, @chriscoffee :)

**Comment by mattstratton** [comment]
@chriscoffee I would like to get back to working on this project, but I've been holding off on anything until this PR is merged. If you aren't planning on doing any more with it, can you document the current state of this, and we can merge it (even though it might have some breaks) and I can pick up from there?

**Comment by chriscoffee** [comment]
Hey @mattstratton. Sorry about this! 😞 I took a break from _basically_ everything...

I think with this ticket it's best I split it down to its individual parts then it's not blocking:
1. The fixes to get modules and up to date go versioning. (This will be relatively easy to pull out, I'll also update stuff again whilst I'm there)
2. Moving some of the deployment stuff out (don't think there's _much_ left here just some discussion around keying)

I'll pull off 1 ASAP (probably try get it done when I get home from work) to get this going then the other can just work on.

Again sorry for the delay!

**Comment by mattstratton** [comment]
Sounds good! I've been seeing a few issues pop up in `devopsdays/devopsdays-web` that are related to the bash scripts which would be solved with `devopsdays-cli` so I was going to get going on this again :)

**Comment by chriscoffee** [comment]
Closing this PR as I'm opening the above on another branch.

Cheers

**Pull request closed without merging**